src/System.Data.SqlClient/ref/System.Data.SqlClient.csproj

```diff
@@ -4,19 +4,27 @@
   <PropertyGroup>
     <ProjectGuid>{D58E8D2B-3331-4660-8DFB-512D66F8EC63}</ProjectGuid>
     <IsPartialFacadeAssembly Condition="'$(TargetGroup)' == 'netfx'">true</IsPartialFacadeAssembly>
-    <!-- Must match version supported by frameworks which support 4.3.* inbox.
-         Can be removed when API is added and this assembly is versioned to 4.4.* -->
-    <AssemblyVersion Condition="'$(TargetsNetFx)' != 'true'">4.3.1.0</AssemblyVersion>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netfx-Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netfx-Release|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard-Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard-Release|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap-Debug|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap-Release|AnyCPU'" />
   <ItemGroup>
     <SuppressPackageTargetFrameworkCompatibility Include="$(UAPvNextTFM)" />
     <Compile Include="System.Data.SqlClient.cs" />
     <Compile Include="System.Data.SqlClient.Manual.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == 'uap'">
```

Comment:
Don'y use project references.  Instead use simple-name references.

EG:
```xml
    <Reference Include="System.Runtime" />
    <Reference Include="System.Runtime.Extensions" />
    <Reference Include="System.Runtime.InteropServices" />
    <Reference Include="System.Data.Common" />
    <Reference Include="System.ComponentModel.Primitives" />
    <Reference Include="System.Xml.ReaderWriter" />
    <Reference Include="System.Collections.NonGeneric" />
```

Reply:
Actually I don't even think a UAP configuration is necessary here.

Reply:
If I remove the uap configs from the ref files, then I get the following error:
CoreFx\bin\ref\System.Data.SqlClient\4.4.0.0\netstandard\System.Data.SqlClient.dll and CoreFx\bin\ref\System.Data.SqlClient\4.4.0.0\uap\System.Data.SqlClient.dll have the same TargetPath Microsoft.Private.CoreFx.UAP/ref/uap10.0.16300/System.Data.SqlClient.dll.

Reply:
I think that's just because you need to do a clean build.